pcsx2/gui/AppMain.cpp

```diff
@@ -529,10 +531,34 @@ extern bool FMVstarted;
 extern bool renderswitch;
 extern bool EnableFMV;
 
-void DoFmvSwitch()
+void DoFmvSwitch(bool on_off)
 {
-	ScopedCoreThreadPause paused_core( new SysExecEvent_SaveSinglePlugin(PluginId_GS) );
-	renderswitch = !renderswitch;
+	ScopedCoreThreadPause paused_core(new SysExecEvent_SaveSinglePlugin(PluginId_GS));
```

Comment:
I had some issues with that when I started out with this PR, so I decided to keep it this way. But I just tested on the current code with it restricted to the software game fix and it works without issue now. I'll update.